ref/gradescope.md

```diff
@@ -83,7 +83,7 @@ Have a productive week!
 <!-- Acknowledgements
 {: .fs-4 }
 
-* Special thanks to Liubov Kurafeeva and Roman Beltiukov for creating the initial instructions and screenshots.
+* Special thanks to Liubov Kurafeeva and Roman Beltiukov for your helps! 🙏🏼
```

Comment:
?

Reply:
@ykharitonova I thought you said to change the exact sentence of  "creating the initial instructions and screenshots." to "your helps!".  It seems like I misread it in the text. Could you pls clarify this? Do you mean putting our names instead of those people's names? Do we also need to place your name instead of Prof. Conrad's and change the quarter to S23 in the acknowledgment ("Updated W23 by P. Conrad")?